lib/logflare/backends/adaptor/webhook_adaptor.ex

```diff
@@ -165,7 +165,8 @@ defmodule Logflare.Backends.Adaptor.WebhookAdaptor do
         end
 
       Client.send(
-        url: config.url,
+        # if a `url_override` key is available in the merged config, use that before falling back to `url`
```

Comment:
could you add docs for this behaviour, even if temporary

Reply:
The stored_config should probably get transformed by the respective backend type instead of getting inserted wholesale, that might allow us to avoid this url_overriding as the runtime config used can differ from the stored config.

Reply:
> The stored_config should probably get transformed by the respective backend type instead of getting inserted wholesale, that might allow us to avoid this url_overriding as the runtime config used can differ from the stored config.

Let's see where the ClickHouse stuff goes in the short-term. I do agree that the bandaid I put in place could be improved though.